src/metrics/index.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const HttpClient = require("@actions/http-client").HttpClient
+const AmbassadorClient = require('../ambassador/api-client');
+const core = require('@actions/core');
+const configure = require('../configure');
+
+const httpClient = new HttpClient();
+const ambassadorClient = new AmbassadorClient(process.env.TELEPRESENCE_API_KEY)
+
+class MetritonClient {
+    static async sendMetricsReport(action){
+        try {
+            const reportDestination = 'https://metriton.datawire.io/scout'
+            const applicationName = 'telepresence-github-action-integration'
+            const installId = fs.readFileSync(`${configure.getTelepresenceConfigPath()}/id`).toString()
+            const extensionVersion = process.env.ACTION_VERSION	
+            const pipelineId = `${process.env.GITHUB_RUN_ID}-${process.env.GITHUB_RUN_ATTEMPT}`
+            const userInfoResponse = await ambassadorClient.doRequestJson('/userinfo', null, 'GET')
+            if (userInfoResponse.statusCode !== 200) {
+                throw new Error('Could not get user information')
+            }
```

Comment:
this should probably be a function inside the `AmbassadorClient` class so we can just invoke a function to get the userinfo instead of having consumers of the AmbassadorClient to know what endpoint to call.